test/spec/testUtils.spec.ts

```diff
@@ -0,0 +1,169 @@
+var utils;
+
+// This needs to be cleaned, and maybe externalized into a small testing
+// framework utils. But then we'd need to find a name for it.
+utils = angular.module('testUtils', []);
```

Comment:
Protractor's way of dealing with events is pretty limited IMHO... It doesn't handle mouseover/mouseout events, for example